person.rb

```diff
@@ -9,6 +11,7 @@ def initialize(age = 10, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super
```

Comment:
- [ ] Kindly remove `super` as it causes an error when I run your app and try to create a new person.
![image](https://user-images.githubusercontent.com/24703171/210812531-45a10e88-5028-42d1-bc24-9ef79b19b6f5.png)

Reply:
Hi @gemmen29, whenever I removed the super, I always had linter errors. So I called it as a function instead and it worked fine. Attached below is a screenshot of my final result. If you could be so kind as to check again or please advise on another way to prevent the linter error, I would be glad.

![2023-01-05_15h21_03](https://user-images.githubusercontent.com/105272189/210815288-dca3ec5a-1c5c-4eac-b82b-614876deed9e.png)